gapidapk/deviceinfo.go

```diff
@@ -144,7 +151,7 @@ func fetchDeviceInfo(ctx context.Context, d adb.Device) error {
 	}
 	defer cleanup.Invoke(ctx)
 
-	if d.Instance().GetConfiguration().GetOS().GetAPIVersion() >= 29 {
+	if gpuProfilingSupported && d.Instance().GetConfiguration().GetOS().GetAPIVersion() >= 29 {
```

Comment:
Maybe we can get rid of the API level check here, since gpuProfilingSupported should suffice?